src/EnergyPlus/DXCoils.cc

```diff
@@ -10407,9 +10409,12 @@ Label50: ;
 		Real64 OutletAirHumRat; // outlet air humidity ratio [kg/kg]
 		Real64 TotCapHS; // total capacity at high speed [W]
```

Comment:
Looking at the 2 equations for InputPowerMult (B) and HeatingCapMult (y), Heating Capacity falls off more quickly (-107) than Power (-36) as delta w increases. This would result in COP decreasing with increasing delta w which makes sense. I think these are the tests needed to resolve which equations get modified (I'm still unsure of PLRHeating). Can we first plot existing capacity/power vs delta w to see what that looks like. Then modify equations and plot again. Each time noting COP.